README.md

```diff
@@ -4,187 +4,86 @@
 
 [![KubeOne Report Card](https://goreportcard.com/badge/github.com/kubermatic/kubeone)](https://goreportcard.com/report/github.com/kubermatic/kubeone)
 
-`kubeone` is a CLI tool and a Go library for installing, managing, and upgrading
-Kubernetes clusters. It can be used on on all your cloud, on-prem, edge, and IoT environments.  
-
-KubeOne can install high-available (HA) master cluster as well single master cluster.
-
-## Project Status
-
-As of v0.6.0, KubeOne is in the beta phase. Check out the
-[Backwards Compatibility Policy][6] for more details on
-backwards compatibility, KubeOne versioning, and maturity of each KubeOne
-component.
-
-Versions earlier than v0.6.0 are considered alpha and it's strongly advised to
-upgrade to the v0.6.0 or newer as soon as possible.
-
-## Getting Started
-
-All user documentation is available at the [KubeOne docs website](https://docs.kubermatic.com/kubeone/master/).
-
-We have a getting started tutorial for each provider we support in the
-[Getting Started section][10]. For example, the following document shows
-[how to get started with KubeOne on AWS][11].
-
-For advanced use cases and other features, check out the
-[Using KubeOne section][13].
-
-## KubeOne in Action
-
-[![KubeOne Demo asciicast](https://asciinema.org/a/244104.svg)](https://asciinema.org/a/244104)
+Kubermatic KubeOne automates cluster operations on all your cloud, on-prem,
+edge, and IoT environments. KubeOne can install high-available (HA) master
+clusters as well single master clusters.
 
 ## Features
 
-* Supports Kubernetes 1.13+ High-Available (HA) clusters
-* Uses `kubeadm` to provision clusters
-* Comes with a straightforward and easy to use CLI
-* Choice of Linux distributions between Ubuntu, CentOS and CoreOS/Flatcar
-* Integrates with [Cluster-API][7] and [Kubermatic machine-controller][8] to
-  manage worker nodes
-* Integrates with Terraform for sourcing data about infrastructure and control
-  plane nodes
-* Officially supports AWS, DigitalOcean, GCE, Hetzner, Packet, OpenStack, VMware
-  vSphere and Azure
-
-## Installing KubeOne
-
-### Downloading a binary from GitHub Releases
-
-The fastest way to get KubeOne:
-```bash
-curl -sfL get.kubeone.io | sh
-```
```

Comment:
Probably we need to save this

Reply:
Do we really need it if we pointed to the documentation a bit down?

Reply:
Yeah, it's for the marketing reasons

Reply:
@kron4eg Fixed, PTAL.